recipe/meta.yaml

```diff
@@ -10,13 +10,11 @@ source:
   sha256: a737521126efc3295b818acc951a745d7b914446836c7931b0b52998f3934e2d
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  skip: true  # [not win]
```

Comment:
Does this need to be modified to build non-windows versions as well. I only see the Azure windows build.

Reply:
Yes, I'm just testing the windows part right now.  (I didn't want to waste a bunch of resources on the other builds).  @jdhughes-usgs do you know if the meson build for intel fortran is statically linked, or does it need the intel fortran runtime dlls?